assetsTransformer.js

```diff
@@ -0,0 +1,7 @@
+const path = require('path');
+
+module.exports = {
```

Comment:
All these changed already in DEV, I've cherry-oicked them into my PR with tests that has been merged already